src/ado2gh/Commands/CreateTeamCommand.cs

```diff
@@ -55,9 +55,16 @@ public async Task Invoke(string githubOrg, string teamName, string idpGroup, boo
 
             var githubApi = _githubApiFactory.Create();
 
-            await githubApi.CreateTeam(githubOrg, teamName);
-
-            _log.LogSuccess("Successfully created team");
+            var teams = await githubApi.GetTeams(githubOrg);
+            if (teams.Contains(teamName))
+            {
+                _log.LogSuccess($"Team '{teamName}' already exists. New team will not be created");
```

Comment:
nit: don't we need a full stop at the end of the second sentence?

Reply:
Looks like we are not using full stops anywhere else. I am modifying it to `-`